PlayerKit/Classes/RegularPlayer.swift

```diff
@@ -64,14 +72,8 @@ extension AVMediaSelectionOption: TextTrackMetadata {
         }
     }
     
-    public let view: UIView = RegularPlayerView(frame: .zero)
-    
-    private var regularPlayerView: RegularPlayerView {
-        return self.view as! RegularPlayerView
-    }
-    
-    private var playerLayer: AVPlayerLayer {
-        return self.regularPlayerView.playerLayer
+    open var view: UIView {
+        return regularPlayerView
```

Comment:
It pains me to say this but for the sake of consistency maybe we slap a `self.` on the front there.